frontend/src/index.css

```
@@ -108,54 +130,50 @@ html {
-
-.table-wrapper th {
-  background-color: rgb(var(--muted));
-  color: rgb(var(--fg));
-}
-
-.table-wrapper td,
-.table-wrapper th {
   padding: 8px 10px;
   vertical-align: middle;
+
+  width: auto !important;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)